packages/jumpstarter-driver-ble/README.md

```
@@ -0,0 +1,60 @@
+## API Reference
+
+```{eval-rst}
+.. autoclass:: jumpstarter_driver_ble.client.BleClient()
+    :members:
+```
+
+### CLI
+
+The ble driver client comes with a CLI tool that can be used to interact with
+the target device.
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)